pkg/healthcheck/healthcheck.go

```diff
@@ -1038,10 +1038,11 @@ func (hc *HealthChecker) allCategories() []category {
 					},
 				},
 				{
-					description:   "control plane self-check",
-					hintAnchor:    "l5d-api-control-api",
-					fatal:         true,
-					retryDeadline: hc.RetryDeadline,
+					description:         "control plane self-check",
+					hintAnchor:          "l5d-api-control-api",
+					surfaceErrorOnRetry: false,
```

Comment:
Any reason this has to be false? As its hard to understand which check we are waiting for with just `\ waiting for check to complete`?

Reply:
I get the following output, by making it `true`
```
- Error calling Prometheus from the control plane: server_error: server error: 503
```
This seems better than the waiting for check right?

Reply:
As you know, prometheus takes a long while to become available, so users that run `linker check` right after install will always see that error. I thought it was less scary for them to just let them know we're waiting for the check. If prometheus doesn't come up by the time-out, the real error is shown:
```
linkerd-api
-----------
√ control plane pods are ready
√ control plane self-check
√ [kubernetes] control plane can talk to Kubernetes
× [prometheus] control plane can talk to Prometheus
    Error calling Prometheus from the control plane: server_error: server error: 503
    see https://linkerd.io/checks/#l5d-api-control-api for hints

Status check results are ×
```